web/src/components/users/UserEditForm.vue

```diff
@@ -71,6 +71,7 @@
           :model-value="groupMembershipAttributes.departmentId"
           :type="UserGroupTypes.DEPARTMENT"
           :parent-id="null"
+          :disabled="!isSystemAdmin"
```

Comment:
Due to how Vuetify defaults look - disabled is basically unreadable due to low contrast - we've been avoiding "disabled" and using "readonly" + `append-inner-icon="mdi-lock"`